runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -1173,6 +1173,83 @@ NNFW_STATUS nnfw_session::set_backends_per_operation(const char *backend_setting
   return NNFW_STATUS_NO_ERROR;
 }
 
+NNFW_STATUS nnfw_session::train_get_traininfo(nnfw_train_info *info)
+{
```

Comment:
Please add null pointer check code for `info`

Reply:
updated :)